### DISCUSSION TITLE
[DOMPDF] LIMITE DE PÁGINAS AL GENERAR PDF

### DISCUSSION THREAD

**Discussion by LeandAyala** [community/community#27324] [Other Feature Feedback, Questions, & Ideas]
Hola amigos, primeramente quiero decirles que Dompdf es una librería maravillosa. No se si este sea el espacio correcto, pero tengo una duda y un problema a la vez: sucede que estoy generando un pdf que ocupa mas de 100 hojas, pero al parecer existe un límite de 45, el cual provoca que no se genere el pdf si se excede esta cantidad. ¿Cómo puedo solucionar este aspecto? Muchas gracias.

**Comment by rromulos** [comment]
Could you tell which error are you seeing in the log? Maybe problem with memory exhausted?
